website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta.md

```diff
@@ -17,45 +20,88 @@ dbt Cloud Enterprise supports single-sign on via Okta (using SAML). Currently su
-On the *General Settings* page, enter the following:
+### Configure the Okta application
+
+On the **General Settings** page, enter the following details::
 
 * **App name**: dbt Cloud
-* **App logo** (optional): You can optionally [download this dbt logo](https://raw.githubusercontent.com/fishtown-analytics/corp/master/assets/dbt/dbt-logo-75x75.png), and upload it to Okta to use as the logo for this app.
+* **App logo** (optional): You can optionally [download the dbt logo](https://drive.google.com/file/d/1w_Yj7QK-ULP4ebtKbrbvGo04pWlg0Y7S/view),
+  and upload it to Okta to use as the logo for this app.
 
 Click **Next** to continue.
 
-<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/ae2045a-Screen_Shot_2019-04-25_at_6.05.12_PM.png" title="The 'General Settings' page"/>
+<Lightbox
+    collapsed={false}
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/okta/okta-2-general-settings.png"
+    title="Configure the app's General Settings"
+/>
 
 ### Configure SAML Settings
 
-:::caution Group Attributes
-
-dbt Cloud uses SAML settings provided by Okta to enforce role-based access control. If the Group Attribute statement shown below is misconfigured, then users may not be permissioned to projects appropriately in dbt Cloud.
+The SAML Settings page configures how Okta and dbt Cloud communicate. If your
+dbt Cloud instance is _not_ running at `cloud.getdbt.com`, you will want to replace
+the domain names shown below with the domain name where your instance is running. If you
+aren't sure which values you should use, please contact support (support@getdbt.com).
 
-:::
+To complete this section, you will need a _login slug_. This slug controls the
+URL where users on your account can log into your application via Okta. Login
+slugs are typically the lowercased name of your organization separated with
+dashes. For example, the _login slug_ for Fishtown Analytics would be
+`fishtown-analytics`. Login slugs must be unique across all dbt Cloud accounts,
+so pick a slug that uniquely identifies your company.
 
 On the **SAML Settings** page, enter the following values:
 
-* **Single sign on URL**: `https://auth.getdbt.com/login/callback?connection=<your-deployment-id>`
-* **Audience URI (SP Entity ID)**: `urn:auth0:dbt-cloud:<your-deployment-id>`
+* **Single sign on URL**: `https://cloud.getdbt.com/complete/okta`
+* **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`
+* **Relay State**: `<login slug>`
 
-Replace `<your-deployment-id>` with your dbt Cloud deployment ID. If you aren't sure what value you should use, please contact support (support@getdbt.com).
+<Lightbox
+    collapsed={false}
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/okta/okta-3-saml-settings-top.png"
+    title="Configure the app's SAML Settings"
+/>
 
-<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/61d2b48-Screen_Shot_2019-04-25_at_6.05.23_PM.png" title="The 'SAML Settings' page"/>
+<!-- TODO : Will users need to change the Name ID format and Application
```

Comment:
@kwigley would love to hear your thoughts on this part in particular. I saw an example where you set these fields to `Email Address` explicitly. Is that a configuration that we should document here?